examples/api/cpp/finite_field.cpp

```diff
@@ -0,0 +1,63 @@
+/******************************************************************************
+ * Top contributors (to current version):
+ *   Alex Ozdemir
+ *
+ * This file is part of the cvc5 project.
+ *
+ * Copyright (c) 2009-2022 by the authors listed in the file AUTHORS
+ * in the top-level source directory and their institutional affiliations.
+ * All rights reserved.  See the file COPYING in the top-level source
+ * directory for licensing information.
+ * ****************************************************************************
+ *
+ * An example of solving finite field problems with cvc5's cpp API
+ */
+
```

Comment:
This example is broken without configuring with `--cocoa`. Should I guard it? Same for the Python API example.

Reply:
Yep! I've guarded all of the examples and tests.